pkg/epinio/l10n/en-us.yaml

```diff
@@ -130,6 +130,15 @@ epinio:
         title: Routes
         tooltip: Replace the default route (<app name>.<epinio domain>) with one or more custom routes
         placeholder: e.g. my-custom-route.com/my-app
+    export:
+      label: Export App
+      manifest:
+        title: Export Manifest
+        description: Export the manifest for this application
```

Comment:
Need to explain what a manifest is.

Something like `Manifests contain basic application configuration like name, routes, number of instances, etc).